datafusion/expr/src/logical_plan/display.rs

```diff
@@ -181,7 +181,7 @@ impl<'a, 'b> GraphvizVisitor<'a, 'b> {
     }
 }
 
-impl<'n, 'a, 'b> TreeNodeVisitor<'n> for GraphvizVisitor<'a, 'b> {
```

Comment:
the underscore does it mean unused lifetime?

Reply:
https://doc.rust-lang.org/reference/lifetime-elision.html